builtin/logical/pki/path_config_urls.go

```diff
@@ -120,29 +131,65 @@ func (b *backend) pathWriteURL(ctx context.Context, req *logical.Request, data *
 		return nil, err
 	}
 
+	if enableTemplating, ok := data.GetOk("enable_templating"); ok {
+		entries.EnableTemplating = enableTemplating.(bool)
```

Comment:
A thought, if we go from true to false on the EnableTemplating here, should we verify that the various urls do not have the replacement strings within them? I believe we will catch `{{cluster_path}}` but not `{{issuer_id}}` in the validation below.

Reply:
Or possibly in general add that validation if `EnableTemplating` is false?

Reply:
Okie dokie, updated!